### PR TITLE
depends: Exclude more unix/Linux graphical interface libraries

### PR DESCRIPTION
**Pull request by Jmennius**
These usually come with the graphical stack,
including them may cause unexpected issues.

First regex picks up everything that comes out of `libglvnd`
(https://github.com/NVIDIA/libglvnd,
https://packages.ubuntu.com/source/focal/libglvnd),
a total of 7 libraries.
The second one is for DRM Linux kernel interface library.

Partially-fixes: #4951

P.S. Please note that I do not have a deep understanding of the graphical stack, this is my best effort.

### CODE REVIEW

**Comment by Legorooj** [comment]
@Jmennius I'm fairly certain that #5058 is the cause of this, so I'm going to have to hold off on this PR for now.

**Comment by Jmennius** [comment]
> @Jmennius I'm fairly certain that #5058 is the cause of this, so I'm going to have to hold off on this PR for now.

It doesn't look related to me (#5058 is about data files, not binaries + I am using v3.6 for now and the regression #5058 is about was introduced in v4.0?), but take your time to understand what is going on, I am not pushing :)

**Comment by Legorooj** [comment]
@Jmennius the line between data files and boundaries is very thin in some cases - like this. #5058 is causing extra files for `gi` to be included, which is causing 1gb output binaries, 🤷. Better safe than sorry I say in this case.

**Comment by Jmennius** [comment]
@Legorooj so what do you say - should we proceed with this now?

**Comment by Legorooj** [comment]
@Jmennius yes, we can merge this if I can get an approving review from @rokm who knows Linux better than I do.

**Review by rokm — APPROVED** [review]
The additions to exclude list look good to me.

The driver-stack specific OpenGL shared libraries (NVIDIA, mesa) are probably not directly linked these days thanks to `libglvnd`, but it makes sense to use target system's `libglvnd` libraries instead of collecting the ones from the build system. (I guess if build system uses `libglvnd` but the target system does not, there will be other, bigger issues preventing the use of program anyway).

And I think `libdrm` should definitely be excluded. The actual DRM drivers are linked against it and we do not collect those, so mixing the target system's drm drivers and build system's `libdrm` can be problematic.

**Comment by Legorooj** [comment]
Thanks @Jmennius

**Pull request merged**
